### ISSUE TITLE
Back button should close zendesk window on mobile

### ISSUE THREAD

**Issue by noisecapella** [mitodl/micromasters#1516]
On mobile the zendesk widget will take up the entire screen. If the user clicks the back button at that point it will go back to the previous point but the zendesk widget will remain open. It looks like nothing changed. We could just close the zendesk widget window whenever the URL changes, in the same function we use to trigger google analytics events

**Comment by pdpinch** [comment]
Is this exclusively a mobile problem?

**Comment by noisecapella** [comment]
Yes, on desktop the zendesk widget takes up a small part of the screen so it doesn't obscure the fact that the browser contents underneath changed. However I feel like closing the zendesk widget for any change of URL would be reasonable to do both on mobile and desktop
